test/simple_test/simple_test.c

```diff
@@ -80,35 +84,43 @@ main(int argc, char *argv[])
 
    img = dmtxImageCreate(pxl, width, height, DmtxPack24bppRGB);
    assert(img != NULL);
+   
 
    dec = dmtxDecodeCreate(img, 1);
    assert(dec != NULL);
 
    reg = dmtxRegionFindNext(dec, NULL);
-   if(reg != NULL) {
+   if (reg != NULL)
+   {
       msg = dmtxDecodeMatrixRegion(dec, reg, DmtxUndefined);
 
-      fprintf(stdout, "msg->arraySize :  \"%zd\"\n", msg->arraySize );
-      fprintf(stdout, "msg->codeSize  :  \"%zd\"\n", msg->codeSize  );
+      fprintf(stdout, "msg->arraySize :  \"%zd\"\n", msg->arraySize);
+      fprintf(stdout, "msg->codeSize  :  \"%zd\"\n", msg->codeSize);
       fprintf(stdout, "msg->outputSize:  \"%zd\"\n", msg->outputSize);
+      fprintf(stdout, "msg->fnc1:  \"%zd\"\n", msg->fnc1);
       int oned = sqrt(msg->arraySize);
-      for (int i=0; i<msg->arraySize; i++){
+      for (int i = 0; i < msg->arraySize; i++)
+      {
          fprintf(stdout, " %c.", msg->array[i]);
-         if (i%oned==oned-1){
+         if (i % oned == oned - 1)
+         {
             fprintf(stdout, "\n");
          }
       }
       fprintf(stdout, "\n\n");
-      for (int j=0; j<msg->codeSize; j++){
+      for (int j = 0; j < msg->codeSize; j++)
+      {
          fprintf(stdout, " %c.", msg->code[j]);
       }
       fprintf(stdout, "\n\n");
-      for (int k=0; k<msg->outputSize; k++){
+      for (int k = 0; k < msg->outputSize; k++)
+      {
          fprintf(stdout, " %c.", msg->output[k]);
       }
       fprintf(stdout, "\n\n");
 
-      if(msg != NULL) {
+      if (msg != NULL)
+      {
```

Comment:
also, can you, please, split formatting changes from logic changes to separate pull requests?

// or maybe just set up IDE to use formatting similar to original :shrug:
For example, I'd prefer a formatting scheme with `{` on the same line as the condition it corresponds to, prepended by space (I see thete is lines without spaces, an I can't remember why (may be it is artifacts of previous pull requests, or of previous maintainers), but still I'd like when it on the same line).

Anyway, it would be nice to have them (any formatting changes) in separate PR.